### PR TITLE
ui: Bump node to v14

### PR DESCRIPTION
**Pull request by johncowen**
Bumps node to v14:

1. In nvm for dev work
2. In CI
3. During `make ui` via our docker build file (also bumped alpine to 3.13 and a couple of other deps in there)

### CODE REVIEW

**Review by alvin-huang — APPROVED** [review]

**Review by kaxcode — APPROVED** [review]
👍

**Comment by johncowen** [comment]
It's 'most probably' safe, but it's one of those things that you won't notice if it's not safe until you've used it for a while, which is why I thought it would be good to do it now for master/1.11 features.

I can't think of anything we'd gain from putting this on 1.10 also but we can do if you want?

File: ui/packages/consul-ui/docs/upgrades.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Node
+
+Node upgrades should be done when it is convienient, preferably using the
+latest Active LTS version (https://nodejs.org/en/about/releases/) that also
+corresponds with the ember version we are on (https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md)
+
+Aswell as bumping our `.nvmrc` and `.circleci/config.yml` files, when bumping
+node versions, please check with the rest of the team to see if there are
+other repositories that should track the same node version as this repository.
```
Comment (johncowen) [inline ui/packages/consul-ui/docs/upgrades.mdx:11]: I added some docs in here also

**Comment by johncowen** [comment]
@alvin-huang in the end we decided to backport this to 1.10 also, not sure what we'd need to do elsewhere to reflect that?

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/370120.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 8301e79c5675e753c41e84861b3464cdc673788f onto `release/1.10.x` succeeded!

**Pull request merged**